### DISCUSSION TITLE
Why Are Paths Not Available with Automation API

### DISCUSSION THREAD

**Discussion by aeekayy** [pulumi/pulumi#10945] [Automation API] [answered]
I'm using Pulumi's Automation API to migrate my organization's Ansible playbooks over to Pulumi. In the process of digging into Pulumi, I want to set leverage [structured configuration](https://www.pulumi.com/docs/intro/concepts/config/#structured-configuration). I thought this would be helpful in the cases where I had `n` resources that I want to deploy as part of a stack. I tried to leverage the set config method for my stack. However, I wasn't able to take advantage of paths. I looked at the SDK and I noticed that the `--path` argument is left out of the options for set config. Is there a reason for this? I have two code blocks that show you what I'm talking about.  

Golang code block: https://github.com/pulumi/pulumi/blob/master/sdk/go/auto/local_workspace.go#L158-L169
Python code block: https://github.com/pulumi/pulumi/blob/master/sdk/python/lib/pulumi/automation/_local_workspace.py#L219-L231

**Comment by Frassle** [comment, marked as answer]
We have an issue tracking this: https://github.com/pulumi/pulumi/issues/5506

We also have some internal thoughts to re-design automation api in such a way that stuff like this can never slip unsupported again. Hoping to do some of the work towards that this quarter.
